### ISSUE TITLE
[EN][Quest][Thousand Needles] Release Heartrazor (Minor, Cosmetic)

### ISSUE THREAD

**Issue by Enay** [Atlantiss/BugTracker#3108]
In the quest **Release Heartrazor** (http://www.wowhead.com/quest=28088), **Heartrazor** (http://www.wowhead.com/npc=47486) is missing, there are just 4 NPCs that are supposed to be holding him, but he's not there, you do however complete the quest once you kill all 4 of the NPCs.

How it should work is that **Heartrazor** should be held by these 4 NPCs, as shown on the picture below, and after you've killed all 4 of the NPCs, he should fly off, and go down to where the quest giver is.

Here, though, Heartrazor is always present at the quest giver, even before you finish this quest.

![198089](https://cloud.githubusercontent.com/assets/9798968/7333196/af700c92-eb65-11e4-8d6b-a53488efd7af.jpg)

I'm pretty sure that he used to be there, when I leveled my first character, and he even flew off, after you killed the 4 NPCs, but it seems like it got bugged in the meantime. However, even before, he was always present at the quest giver, even before you finished this quest.

**Comment by Varchord** [comment]
I'm not sure if it's the same issue but you are mentioning something about his absence
https://github.com/Atlantiss/BugTracker/issues/381

**Comment by Enay** [comment]
Yes, I know. This is different though, that time, and it was on the alliance side, he didn't appear at all, and when I circled around with a boat, and went to his location on the top side, he was there, and once I flew with him from up there, he THEN appeared at the NPC that gave the quests. However this time, he was always there, at the NPC, it's on the horde side, though, not sure if that has any impact on this, because it's the same quest for both sides. Anyway, when I did it on my first character, an alliance character, I'm pretty sure that he was actually there, like on the picture in my report, and that he did actually fly off, but he never appeared at the NPC then. This time, though, he wasn't there at all, captured, it was just the 4 npcs, you kill them, and you get a complete quest. He is there at the NPC for flying at all times, though, but he did phase out a few times, and phased back in.

Anyway, every quest can be done, but it's definitely buggy, and not blizzlike.
